### ISSUE TITLE
Add an Audio Player at Bottom of the screen

### ISSUE THREAD

**Issue by sufiishq-pk** [sufiishq/web-app#9]
**Is your feature request related to a problem? Please describe.**
Add an audio player at the bottom of the screen.

**Requirements**
- [ ] By default a player will select a first kalam as a default from the database.
- [ ] Once a kalam is complete then the player would be stopped.
- [ ] Show an elegant loader when a player is loading a kalam.
- [ ] Seekbar should be disabled while a player is loading a kalam.
- [ ] Show total time and elapsed time in a player with this `hh:mm:ss` format.
- [ ] Volume icon button would be worked as a toggle button i.e mute / unmute.
- [ ] Do not add any functionality on `add to playlist` icon and `add to favorite` icon button for the functionality separate ticket will be used.

**Additional context**
- The implementation of controlling the favorite functionality will be tracked here #16 
- The implementation of adding a kalam in the playlist will be tracked here #14 

**Design**

![Player](https://user-images.githubusercontent.com/125591377/227790085-d7114559-3f24-416e-8e80-4c14cede8e21.png)

Resources could be found here https://www.figma.com/file/7gtvtRyA8LgRxuVNotVnOR/SufiIshq-Web-App?node-id=0%3A1&t=YMxLHDBLxlFzZ6Fu-1
